hack/boilerplate.go.txt

```diff
@@ -0,0 +1,6 @@
+/*
```

Comment:
Does it make sense to use a separate file here rather than re-use NOTICE.go.txt

Reply:
Yep, totes missed the NOTICE.go.txt, updated.